src/hermes/commands/harvest/codemeta.py

```diff
@@ -0,0 +1,56 @@
+import glob
+import json
+import pathlib
+import typing as t
+
+import click
+
+from hermes.model.context import HermesHarvestContext
+from hermes.model.errors import HermesValidationError
+
+
+def harvest_codemeta(click_ctx: click.Context, ctx: HermesHarvestContext):
+    """
+    Implementation of a harvester that provides data from a codemeta.json file format.
+
+    :param click_ctx: Click context that this command was run inside (might be used to extract command line arguments).
+    :param ctx: The harvesting context that should contain the provided metadata.
+    """
+    # Get the parent context (every subcommand has its own context with the main click context as parent)
+    parent_ctx = click_ctx.parent
+    if parent_ctx is None:
+        raise RuntimeError('No parent context!')
+    path = parent_ctx.params['path']
```

Comment:
This is kind of repeating...

Reply:
@led02 You mean across harvesters (same code is in the CFF harvester). I can refactor this into either `__init__` or create a superclass for harvesters in general...